### DISCUSSION TITLE
NoRecoil CS2 v2

### DISCUSSION THREAD

**Discussion by McDaived** [McDaived/NoRecoil-CS2#5] [General]
## ![](https://github.com/McDaived/NoRecoil-CS2/assets/18085492/fdee8c61-c0f7-41a2-80a0-15c1b5f5bb95)Important :
- **Make sure add the same mouse sensitivity in the game to gui window and press save .**
- **you can change keybinds values in gui window for weapons as you wish.**
- **Make sure use AutoHotKey V1.x.x**

## ![](https://github.com/McDaived/Discord-Profile-Card/assets/18085492/952742cf-9744-4ccb-9de1-766560ebae12)Features :
- M4A4 (Numpad1)
- M4A1 (Numpad2)
- FAMAS (Numpad3)
- AK47 (Numpad4)
- Galil (Numpad5)
- UMO (Numpad6)
- SG (To use NoRecoil in zoom, hold LAlt key when shooting) - (Numpad7)
- AUG (To use NoRecoil in zoom, hold LAlt key when shooting) - (Numpad8)
- NoRecoil Off (Numpad9)
- Pause/Resume (xButton1) mouse side buttons.  ${\color{red}NEW}$
- Exit (End) ${\color{red}NEW}$
- ~Bhop (Hold Space)~

## ![](https://github.com/McDaived/NoRecoil-CS2/assets/18085492/7eab67ab-4b44-40ee-b050-53e48a856fc5)How to use :
1. Download AutoHotKey **V1** and install it.
2. Extract zip folder of NoRecoil.
3. Open ``NoRecoil.ahk``.
4. press on Numpad keys to turn on NoRecoil or you can change keybinds values in gui window for weapons as you wish.
5. done.

## ![](https://github.com/McDaived/NoRecoil-CS2/assets/18085492/54cde871-7247-4d88-be81-ffa0ea3173aa)Change log :
```diff
v2:
+ The recoil has been improved and make it a little smoother.
+ gui window to edit value of hotkey
+ Add Pause/Resume key to make it easy use
- Remove Bhop
```

## ![](https://github.com/McDaived/Discord-Profile-Card/assets/18085492/7a4879fd-97a1-4807-98e5-8f62137dee6e)Preview :
![ezgif com-optimize](https://github.com/McDaived/NoRecoil-CS2/assets/18085492/79643cd3-5286-4258-8c5e-ae010a9a2a6e)

<hr /><em>This discussion was created from the release <a href='https://github.com/McDaived/NoRecoil-CS2/releases/tag/v2'>NoRecoil CS2 v2</a>.</em>

**Comment by thinhlx006** [comment]
good one

**Comment by Synx400** [comment]
Bro Recoils are still not accurate. first 15 bullets not going Straight on (AK, M4A4, FAMAS, GALIL) and also need to drag the mouse down a little please make it more accurate. Thanks bro and in my personal opinion you are the best ahk script maker <3

**Comment by flakka2022** [comment]
ping - need update

**Comment by RobertoCostaTupinamba** [comment]
need update, because doesn't work more
